library/src/main/java/com/novoda/downloadmanager/FileDownloaderType.java

```diff
@@ -0,0 +1,24 @@
+package com.novoda.downloadmanager;
+
+import java.security.InvalidParameterException;
+
+enum FileDownloaderType {
+    NETWORK("network"),
+    CUSTOM("custom");
+
+    private final String rawValue;
```

Comment:
Is this even used at this point?
the `from()` method is introduced in this PR but never called... I'd go removing the rawValue entirely if not used